gsa/src/gmp/models/report/host.js

```diff
@@ -97,9 +105,21 @@ class Host {
           case 'best_os_txt':
             copy.details.best_os_txt = value;
             break;
+          case 'traceroute':
+            copy.details.distance = value.split(',').length - 1;
+            break;
+          case 'App':
+            appsCount++;
+            break;
           default:
             break;
         }
+        if (name.startsWith('Auth')) {
+          const authArray = name.split('-');
+          copy.authSuccess[authArray[1].toLowerCase()] =
+            authArray[2] === 'Success' ? YES_VALUE : NO_VALUE;
```

Comment:
Could you do me a favor an not to use YES_VALUE and NO_VALUE for booleans anymore? We should use only true/false now. IMHO using the the values from gvmd/gsad 1:1 was a bad idea :disappointed: 

```suggestion
            authArray[2] === 'Success' ? true : false;
```